### DISCUSSION TITLE
Wallet will not Sync - Windows GUI

### DISCUSSION THREAD

**Discussion by mwebb1508** [Chia-Network/chia-blockchain#12311] [Support]
Hello, I cannot get my wallet to sync. My node is fully synced but the wallet is not. Its stuck at my intial faucet deposit and not seeing any other chia received. It also only shopws onw of my Plot NFTs and gives the error message on that plot of:  You can still create plots for this plot NFT, but you can not make changes until sync is complete.

I have tried the follwing:

Deleted wallet and tried to re-sync, no change.
Fully uninstalled and reinstalled 1.3.5 and 1.4
Uninstalled usined Iobit uninstalled. Deleted all residual files iundluing old DB and Wallet files.
Clean reinstalled windows, Reinstalled chia,  re-syned from 0, still having same error. 

Help!

**Comment by loppefaaret** [comment]
sorry for the late reply, are you still having issues ?
